src/lib/pathPlanner.ts

```diff
@@ -50,6 +51,31 @@ export function isAcyclic(
 	return alg.isAcyclic(graph);
 }
 
+/**
```

Comment:
Add comments:
* For what is it for
* Idea of the approach

Reply:
- Add comments: solved